cpp/models/lct_secir_2_diseases/parameters.h

```diff
@@ -0,0 +1,665 @@
+/* 
+* Copyright (C) 2020-2025 MEmilio
+*
+* Authors: Annika Jungklaus, Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef LCT_SECIR_2_DISEASES_PARAMS_H
+#define LCT_SECIR_2_DISEASES_PARAMS_H
+
+#include "memilio/config.h"
+#include "memilio/utils/parameter_set.h"
+#include "memilio/utils/logging.h"
+#include "memilio/utils/uncertain_value.h"
+#include "memilio/epidemiology/uncertain_matrix.h"
+
+namespace mio
+{
+namespace lsecir2d
+{
+
+/*********************************************************
+* Define Parameters of the LCT-SECIHURD-2-DISEASES model *
+**********************************************************/
+
+/**
+ * @brief Average time spent in the Exposed compartment for disease a in day unit.
```

Comment:
```suggestion
 * @brief Average time spent in the Exposed compartment for disease a in day unit for each group.
```
Same for the other parameters.